dynamics-nav/Sizing-a-Control-Add-in-on-a-Page.md

```diff
@@ -12,16 +13,16 @@ caps.latest.revision: 13
 manager: edupont
 ---
 # Sizing a Control Add-in on a Page
-By default, the size of a control add-in on a page is determined by the [System.Drawing.Size](https://docs.microsoft.com/search/index?dataSource=previousVersions&search=System.Drawing.Size) class of the control. The size is static, so when the page is resized in the [!INCLUDE[nav_windows](includes/nav_windows_md.md)], the control add-in size will remain the same.  
+By default, the size of a control add-in on a page is determined by the [System.Drawing.Size](/search/index?dataSource=previousVersions&search=System.Drawing.Size) class of the control. The size is static, so when the page is resized in the [!INCLUDE[nav_windows](includes/nav_windows_md.md)], the control add-in size will remain the same.  
```

Comment:
It looks like all of these links are broken. You may want to get feedback from Derek or Chong on these before submitting it for review.